09-mapping.Rmd

```diff
@@ -687,11 +720,14 @@ library(globe)
 ```
 
 **tmap** provides a powerful interface for creating a wide range of static maps (section \@ref(interactive-maps) shows how **tmap** also supports interactive maps).
+<!-- if this section is on static map production, it should be part of section static maps -->
 But there are many other options for creating static maps.
```

Comment:
If this section is on static map production, it should be part of section static maps.

Reply:
No, this provides context that (hopefully) makes the rest of the section make sense. I think `plot()` is justified here because when we say 'other' we mean 'other than **tmap**. **sf** and **raster** are legitimate packages for plotting and deserve mention early in this section (also for context) although this is not their focus. Although happy to see alternatives proposed either as a commit directly to master or a PR (preferably).